main.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"io"
+	"log"
+	"net/http"
+	"os"
+)
+
+func main() {
+	file, err := os.Open("paste.html")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
```

Comment:
Technically this is a leak, because the reference is blown away later
It also keeps the file locked.
os.ReadFile is safer for this type of process.